hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerDoubleBuffer.java

```diff
@@ -147,22 +150,29 @@ public Builder setIndexToTerm(Function<Long, Long> termGet) {
       return this;
     }
 
+    public Builder setAvailPendingRequestNum(Semaphore pendingReqNum) {
```

Comment:
It's good to keep the builder as lightweight as possible. Instead of passing the Semaphore, can we pass the maxRequestsInDoubleBuffer as an int parameter and initialize the Semaphore in OzoneManagerDoubleBuffer's constructor.